helm/prometheus-meta-operator/templates/prometheus-rules/aws.workload-cluster.rules.yml

```diff
@@ -92,6 +92,22 @@ spec:
         severity: page
         team: firecracker
         topic: kubernetes
+    - alert: WorkloadClusterHAMasterDownForTooLong
+      annotations:
+        description: '{{`Master node in HA setup is down for a long time.`}}'
+        opsrecipe: master-node-missing/
+      expr: kubernetes_build_info{app="kubelet"} and on(cluster_id) sum(kubernetes_build_info{app="kubelet", role="master"}) by (cluster_id) == 2
+      for: 1h
```

Comment:
I would lower it to 30m, 1h seems pretty long to me 🙂

Reply:
Sure, let's go with 30 minutes.